python/lsst/sims/utils/samplingFunctions.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from lsst.sims.utils import ObservationMetaData
+
+__all__ = ['sample_obsmetadata', 'samplePatchOnSphere']
+
+def sample_obsmetadata(obsmetadata, size=1, seed=1):
+    """
+    Sample a square patch on the sphere overlapping obsmetadata
+    field of view by picking the area enclosed in
+    obsmetadata.pointingRA \pm obsmetadata.boundLength
+    obsmetadata.pointingDec \pm obsmetadata.boundLength
+
+    Parameters
+    ----------
+    obsmetadata: instance of
+        `sims.catalogs.generation.db.ObservationMetaData`
+    size: integer, optional, defaults to 1
+        number of samples
+
+    seed: integer, optional, defaults to 1
+        Random Seed used in generating random values 
+    Returns
+    -------
+    tuple of ravals, decvalues
+    """
+
+    mydict = obsmetadata.summary
+    phi = np.radians(mydict['pointingRA'])
+    theta = np.radians(mydict['pointingDec'])
+    equalrange = np.radians(mydict['boundLength'])
+    ravals, thetavals = samplePatchOnSphere(phi=phi,
+					    theta=theta,
+					    delta=equalrange,
+					    size=size,
+                                            seed=seed)
+    return ravals, thetavals
+
+
+def samplePatchOnSphere(phi, theta, delta, size, seed=1):
```

Comment:
The convention in sims is to take all angle arguments in degrees.  If you want to accept phi, theta in radians, you should put an underscore at the front of the method name.